tests/Doctrine/Tests/ORM/Mapping/BasicInheritanceMappingTest.php

```diff
@@ -250,6 +251,15 @@ public function invalidHierarchyDeclarationClasses(): Generator
         yield 'complex example (Entity Root -> Mapped Superclass -> transient class -> Entity)'
             => [InvalidComplexRoot::class, InvalidComplexEntity::class];
     }
+
+    /** @group DDC-964 */
+    public function testInvalidOverrideFieldInheritedFromEntity(): void
+    {
+        $this->expectDeprecationWithIdentifier('https://github.com/doctrine/orm/pull/10470');
```

Comment:
If this behaves like `expectException`, it should be moved between the 2 lines below, so that triggering the deprecation in `getMetadataFor` doesn't make this test pass.

Reply:
Fixed that.